source/content/email.md

```diff
@@ -122,4 +122,12 @@ Please see Google's help article: [My client isn't accepting my username and pas
 
 Office 365 uses ports `25` and `587` by default, and different options for sending via SMTP client, Direct send or SMTP relay. [This document](https://docs.microsoft.com/en-us/Exchange/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-3) outlines the limitations and configurations of each, to know what will work for your application.
 
-Because we don't support SPF, it is likely that most Exchange or Office 365 servers won't work if its [configured at your email server](https://docs.microsoft.com/en-us/office365/SecurityCompliance/set-up-spf-in-office-365-to-help-prevent-spoofing). Check your organization's Microsoft Exchange settings on what's allowed by your system.
+Because we don't support SPF, it is likely that most Exchange or Office 365 servers won't work if [configured at your email server](https://docs.microsoft.com/en-us/office365/SecurityCompliance/set-up-spf-in-office-365-to-help-prevent-spoofing). Check your organization's Microsoft Exchange settings to see what's allowed by your system.
+
+### Why does my site receive numerous requests to autodiscover.xml?
+
+Autodiscover enables the configuration of Outlook profiles, based only on a username and password. It allows users to automatically connect to their email servers without having to configure the settings manually. An issue can occur when the mail software erroneously and repeatedly calls for an `autodiscover.xml` file. If left unmanaged, this may start using unnecessary resources, return 404 errors, and can result in a slower site.
```

Comment:
This might be TMI for someone trying to solve their issue at hand. We could perhaps edit this down to something like: 

> Autodiscover.xml automates the configuration of Outlook email server authentication.

And/or link off to Microsoft's docs on this: https://docs.microsoft.com/en-us/exchange/architecture/client-access/autodiscover?view=exchserver-2019

Reply:
On this part: 

> An issue can occur when the mail software erroneously and repeatedly calls for an `autodiscover.xml` file.

Add: 

> An issue can occur when the mail software erroneously and repeatedly calls for an `autodiscover.xml` file that does not exist.